spec/system/task_spec.rb

```diff
@@ -1,24 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
+  let(:task_done) { create(:task, :done, project_id: project.id) }
+
   describe 'Task一覧' do
     context '正常系' do
       it '一覧ページにアクセスした場合、Taskが表示されること' do
         # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_tasks_path(project)
         expect(page).to have_content task.title
         expect(Task.count).to eq 1
         expect(current_path).to eq project_tasks_path(project)
       end
 
-      xit 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
+      it 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
         # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_path(project)
         click_link 'View Todos'
+        # 別タブで開くので、別タブを指定
+        windows = page.driver.browser.window_handles
+        page.driver.browser.switch_to.window(windows.last)
```

Comment:
細かいので修正不要ですが、1行でも記載できますね！
```suggestion
        switch_to_window(windows.last)
```

Reply:
ありがとうございます。
動作確認のために試してみたのでそのままコミットしました。[216024c](https://github.com/MocoAristo/rspec_app_exam/pull/1/commits/216024c12bcf9578afbcf3abf342d5f187cf8172)